frontend/src/views/Collection/style.ts

```diff
@@ -67,3 +72,36 @@ export const StyledCallout = styled(Callout)`
   padding: ${PT_GRID_SIZE_PX}px ${PT_GRID_SIZE_PX * 1.5}px;
   margin-bottom: ${PT_GRID_SIZE_PX * 2}px;
 `;
+
+export const ViewCollection = styled.div`
+  padding: 40px;
+`;
+
+export const CollectionHero = styled.div`
+  align-items: flex-start; /* top aligns collection name with action buttons */
+  column-gap: 40px;
+  display: flex;
+
+  /* collection name */
+  h3 {
+    color: ${PT_TEXT_COLOR};
+    flex: 1;
+    letter-spacing: -0.23px;
+    margin: 2.5px 0 0; /* facilitates the center alignment of single-line collection name and top alignment of the first line of a multi-line collection name (line height at 25px) with action buttons (height 30px) */
+  }
+`;
+
+export const CollectionDetail = styled.div`
+  align-items: flex-start;
+  display: grid;
+  grid-template-areas: "description . metadata"; /* grid areas for collection description and metadata with a null cell token (unnamed area) for the allocation of a gutter between the two columns */
```

Comment:
I see the mock has 40px gap, so would the following work?

```ts
export const CollectionDetail = styled.div`
  align-items: flex-start;
  display: grid;
  grid-template-areas: "description metadata"; /* grid areas for collection description and metadata with a null cell token (unnamed area) for the allocation of a gutter between the two columns */
  grid-template-columns: 1fr 1fr; /* grid columns for collection description and metadata (with 1fr allocated to column separation) */
  gap: 40px;
  margin: 16px 0 44px;
`;
```

<img width="1186" alt="Screen Shot 2022-02-09 at 11 14 30 AM" src="https://user-images.githubusercontent.com/6309723/153273675-31db1aea-c6c4-43ab-9688-d5e6bd613b4b.png">